modules/gitops-argo-cd-properties.adoc

```diff
@@ -10,40 +10,101 @@
-|`ResourceExclusions` |__<Object>__|Completely ignore entire classes of resource group.
-|`ResourceInclusions` |__<Object>__|The configuration to configure which resource group/kinds are applied.
-|`Server` |__<Object>__|Argo CD Server configuration options.
-|`SSO` |__<Object>__|Single sign-on options.
-|`StatusBadgeEnabled` |`true` |Enable application status badge.
-|`TLS` |__<Object>__|TLS configuration options.
-|`UserAnonyousEnabled` |`true` |Enable anonymous user access.
-|`Version` |`v2.2.2` |The tag to use with the container image for all Argo CD components.
+|**Name** |**Description** |**Default** | **Properties**
+|`ApplicationInstanceLabelKey` |The `metadata.label` key name where Argo CD injects the app name as a tracking label.|`app.kubernetes.io/instance`|
+|`ApplicationSet` 
+|`ApplicationSet` controller configuration options.
+| _<Object>_
+a|* _<Image>_ - The container image for the ApplicationSet controller. This overrides the **ARGOCD_APPLICATIONSET_IMAGE** environment variable.
+  * _<Version>_ - The tag to use with the ApplicationSet container image.
+  * _<Resources>_ - The container compute resources.
+  * _<LogLevel>_ - The log level used by the ArgoCD Application Controller component. Valid options are debug, info, error, and warn.
+  * _<LogFormat>_ - The log format used by the ArgoCD Application Controller component. Valid options are text or json.
+  * _<PrallelismLimit>_ - The kubectl parallelism limit to set for the controller `(--kubectl-parallelism-limit flag)`.
+|`ConfigManagementPlugins`    |Add a configuration management plugin.| __<empty>__|
+|`Controller`    |Argo CD Application Controller options.|__<Object>__
+a|* _<Processors.Operation>_ - The number of operation processors.
+  * _<Processors.Status>_ - The number of status processors.
+  * _<Resources>_ - The container compute resources.
+  * _<LogLevel>_ - The log level used by the ArgoCD Application Controller component. Valid options are debug, info, error, and warn.
+  * _<AppSync>_ - AppSync is used to control the sync frequency of ArgoCD applications
+  * _<Sharding.enabled>_ - Enable sharding on the ArgoCD Application Controller component. This property is used to manage a large number of clusters to relieve memory pressure on the controller component.
+  * _<Sharding.replicas>_ - The number of replicas that will be used to support sharding of the ArgoCD Application Controller.
+  * _<Env>_ - Environment to set for the application controller workloads.
+|`DisableAdmin`    |Disables the built-in admin user.|`false` |
+|`GATrackingID`    |Use a Google Analytics tracking ID.|__<empty>__|
+|`GAAnonymizeusers`    |Enable hashed usernames sent to google analytics.|`false` |
+|`HA`    |High availablity options.|__<Object>__
+a|* _<Enabled>_ - Toggle high availability support globally for Argo CD.
+  * _<RedisProxyImage>_ - The Redis HAProxy container image. This overrides the ARGOCD_REDIS_HA_PROXY_IMAGE environment variable.
```

Comment:
It would be good to have a consistent formatting for all env variables - some are plain, some bold, some ticked